tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -402,6 +402,168 @@ describe('metrics selectors', () => {
     });
   });
 
+  describe('getMetricsImageCardStepValues', () => {
```

Comment:
All of these tests seems to just be duplicates of the tests on getImageCardStepValues. Why duplicate it? Maybe just test that it calls that function or something? Would be annoying to have to change these tests in 2 places if we want to implement some change in the logic.